connection.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$link= mysqli_connect("localhost","root","1308","okapi_db") or die("connection error");
```

Comment:
Changing the root's password could lead to problems for others unless we change Xampp's configuration files. I would suggest using Ming's way of connecting to SQL server, described in sql_connector.php